### ISSUE TITLE
Add the ability to save the saved pages for offline reading in the internal memory

### ISSUE THREAD

**Issue by AlbahrawyTiger** [kiwibrowser/src.next#157]
Is it possible to make the saved pages for offline reading and save them in a folder in the internal memory instead of saving them within the browser data, as the saved page files can only be accessed by root users.

**Comment by HT-7** [comment]
The **original** Google Chrome actually saves pages into the download folder of the shared internal storage, and also has an option for saving downloads on **MicroSD!** (Benefits: [mass storage access](https://en.wikipedia.org/wiki/Media_Transfer_Protocol#Comparison_with_USB_Mass_Storage) on PC for more convenient file transfer; files not lost if smartphone defective).

I wonder why it was changed for Kiwi browser in first place. Browsers locking saved pages away in `/data`, which is also done by Opera Mini and [Samsung S Browser](https://forum.developer.samsung.com/t/regarding-saved-webpages/5304) last time I checked, has been one of the **long-term annoyances** in mobile web browsing.
This lack of data portability denies backups, meaning without root access, the files will be trapped on that one smartphone indefinitely.

Who thought it would be a smart idea to store saved pages in a locked directory?!
